net-misc/gofish/gofish-1.2.ebuild

```diff
@@ -9,7 +9,7 @@ DESCRIPTION="Gofish gopher server"
 HOMEPAGE="http://gofish.sourceforge.net"
```

Comment:
Could you check, if `https` is supported?

Reply:
It is not :(